.gitignore

```diff
@@ -0,0 +1,11 @@
+./Spring-test/build*
+./Spring-test/bin*
+./Spring-test/.classpath
+./Spring-test/.project
+./Spring-test/.gradle
+./Spring-test/.settings
+Spring-server/.gradle/
+Spring-server/.idea/
+*.iml
+Spring-server/build/
+Spring-server/src/main/resources/public
```

Comment:
Minor, but public resources ignore can be safely removed

Reply:
> Minor, but public resources ignore can be safely removed

Why?

Reply:
Because before I changed bulidsctipt, we copied assets into this path, now we copy them directly into the build folder.
Though there is another question, ideally we do not save resources and config in git, partially because we expose our ssl key, while not important and rather convenient now, we should be more careful if we get a normal signed certificate + key pair.